src/lightning/pytorch/plugins/precision/amp.py

```diff
@@ -100,6 +101,7 @@ def optimizer_step(  # type: ignore[override]
     @override
     def clip_gradients(
         self,
+        module: Optional[Module],
```

Comment:
this would be a breaking change, it has to go to the end of arguments

Reply:
I am saying if user is using positional arguments this will break for him

Reply:
Great point. I've just made the new `module` argument fully optional by listing it as the last optional argument `module: Optional[Module] = None`.